src/locale/locales/en-US.json

```diff
@@ -457,26 +454,27 @@
       "body": "Oops! Could not find any result indicators.",
       "linkText": "Create a result indicator"
     },
+    "noKeyFigures": {
+      "heading": "No key figures",
+      "body": "Oops! Could not find any key figures.",
+      "linkText": "Create a key figure"
+    },
     "itemAdmin": {
       "keyResult": "No key results for selected objective",
       "objective": "No objectives for selected period",
       "period": "No periods"
     },
     "noKPIs": {
-      "heading": "No KPIs",
-      "body": "Oops! Could not find any KPIs.",
-      "adminBody": "Add KPIs that automatically fetch data from Google Sheets.",
-      "linkText": "Create a KPI"
+      "heading": "No measurements",
+      "body": "Oops! Could not find any measurements.",
+      "adminBody": "Add measurement that automatically fetch data from Google Sheets.",
```

Comment:
"Add measurements that automatically fetch [...]" eller "Add measurement that automatically fetches [...]".